tensorflow/lite/testing/generate_examples_lib.py

```diff
@@ -747,6 +765,35 @@ def build_inputs(parameters, sess, inputs, outputs):
   make_zip_of_tests(options, test_parameters, build_graph, build_inputs)
 
 
+@register_make_test_function()
+def make_isfinite_tests(options):
+  """Make a set of tests to do (float) tf.is_finite"""
+
+  test_parameters = [
+      {
+          "input_shape": [[], [1], [2, 3], [1, 1, 1, 1], [1, 3, 4, 3],
+                          [3, 15, 14, 3], [3, 1, 2, 4, 6], [2, 2, 3, 4, 5, 6]],
+      },
+  ]
+
+  def build_graph(parameters):
+    """Build the graph for the test case."""
+
+    input_tensor = tf.placeholder(
```

Comment:
or you can just build a graph like:

input1 = tf.constant(shape=..., dtype=...)
input2 = tf.constant(shape=..., dtype=...)
out = tf.is_finite(input1 / input2)

then when you feed in values, input2 should contain some zeros.

Reply:
@renjie-liu , I have taken your first suggestion and modified the code as per that.

Regards
Amit